assignments/this.js

```diff
@@ -1,26 +1,63 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window Binding -- default binding of the this keyword to the window or DOM, the biggest element in the browser.
+* 2. Implicit Binding -- imply this winthin an object to the object itself. or this reffereing to the block of code the this keyword iis in.
+* 3. Explicit Binding -- asigning the value of this to an object using .call .apply and .bind
+* 4. new Binding -- binding to a new object within a constructor function.
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
 
 // code example for Window Binding
+function world(){
+  "use strict"
+  console.log(this)
+}
 
+world();
 // Principle 2
 
 // code example for Implicit Binding
-
-// Principle 3
-
-// code example for New Binding
-
-// Principle 4
-
-// code example for Explicit Binding
\ No newline at end of file
+const obj = {
+      name: 'someThing',
+      food: 'apple',
+      find: function(){
+        console.log(`${this.name} found ${this.food}.`);
+      }
+    }
+    
+    obj.find();
+// Principle 3 new
+
+function Man(name){
+    this.name = name;
+    this.weapon = 'Fists';
+    this.speak = function(){
+      console.log(`He uses his ${this.weapon}. They call him ${this.name}.`);
+    }
+  }
```

Comment:
There's a lot of weird, inconsistent indentation. Use [prettier](https://prettier.io)!